megatron/data/Makefile

```diff
@@ -1,3 +1,9 @@
+PYTHON3CONFIG := $(shell command -v python3-config 2> /dev/null)
+
+ifndef PYTHON3CONFIG
```

Comment:
should it be `ifndef` or 
```bash
ifeq($(PYTHON3CONFIG),)
endif
```
I think the variable is always defined even if the shell command fails

Reply:
This made the code start working for me, so I don’t think the variable is always defined